test/docs/OpenJ9TestUserGuide.md

```diff
@@ -262,6 +262,15 @@ target
     ```
     element that you want to exclude.
 
+    For tests disabeld with playlist.xml, a exclude list can be generated in 'TestConfig/disabled.mk' 
+    after running 'run_configure.mk' file under 'TestConfig' folder. These tests can be executed through
+    specifying the test target or add `disabled` in front of regular target.
```

Comment:
For tests that are disabled within playlist.xml, `disabled.mk` is generated when running `make -f run_configure.mk`. It contains a list of disabled test targets. These targets can be executed through ...